### DISCUSSION TITLE
How to overwrite the image address of juicedata/mount:ce-v1.2.1 when using sidecar mode ?

### DISCUSSION THREAD

**Discussion by andycountingstars** [juicedata/juicefs#5300] [Q&A]
The address is not accessible, and i want to replace it with a private docker registry address. Pls help !
<img width="993" alt="image" src="https://github.com/user-attachments/assets/6ab103a9-7aed-4cd1-954a-65d9d40168df">
<img width="991" alt="image" src="https://github.com/user-attachments/assets/eadb520a-8d62-4c07-a749-a5bd28a15fc0">

**Comment by zxh326** [comment]
please refer [custom image](https://juicefs.com/docs/csi/guide/custom-image/) docs
